core/src/main/java/org/elasticsearch/search/suggest/SuggestBuilder.java

```diff
@@ -110,6 +114,9 @@ public SuggestionBuilder(String name, String suggester) {
             this.suggester = suggester;
         }
 
+        // for use when constructing builders from an input stream
+        protected SuggestionBuilder() { }
+
```

Comment:
You're right, its a bad hack, I'd like to get rid of it.  I was trying to find a way to create and populate the builder without first knowing the name.  I may need to break it into separate calls.